simple_history/models.py

```diff
@@ -152,7 +153,10 @@ def create_history_model(self, model, inherited):
         attrs.update(Meta=type(str('Meta'), (), self.get_meta_options(model)))
         if self.table_name is not None:
             attrs['Meta'].db_table = self.table_name
-        name = 'Historical%s' % model._meta.object_name
+        if self.custom_model_name:
```

Comment:
Could make this tertiary:
```suggestion
       name = self.custom_model_name if self.custom_model_name is not None else 'Historical%s' % model._meta.object_name
```